### DISCUSSION TITLE
is it possible to migrate to mongodb to supabase?

### DISCUSSION THREAD

**Discussion by lindamacdonald** [supabase/supabase#1092] [Questions]
Hey there guys :)
I have a next.js blog and am using mongodb and  I was wondering if it is possible to migrate to supabase?

thanks in advance

**Comment by silentworks** [comment]
This is possible but it will require you to create tables in supabase that match your collection structure in mongodb. You might also need to think about this a bit because you could potentially optimise this by restructuring your collections when moving to tables.

**Comment by atnegrete** [comment]
Definitely! The real question is, how much do you want to convert from document based to relational? But, either way, it's possible it's just a matter of how much time you can invest into that conversion, in order to optimize as much as possible and decrease duplicated data.

**Comment by lindamacdonald** [comment]
I definately want to make the move🙂 so ill give it a whirl

tor. 8. apr. 2021, 15:36 skrev anegrete ***@***.***>:

**Comment by shubhjhawar** [comment]
I have a django project and my data is stored in mongoDB, now i want to shirt to supabase.
Can anyone tell me the process and walk me through it, line by line?
Thank you

**Reply by anishkumar127** [reply]
Hey,
I have mongodb data and i want to migrate to postgresql please give me step how i can do it in nodejs.
